### ISSUE TITLE
php artisan jwt:generate is not replacing the default value

### ISSUE THREAD

**Issue by Mahmoudz** [tymondesigns/jwt-auth#686]
in my published config file I have this: 

``` php
'secret' => env('JWT_SECRET', 'defaultSecret!'),
```

and in my `.env` file I have this

``` php
JWT_SECRET=xXxXxXxXxXxXxXxXxXxXxXxXxXxXxXxX
```

when I run 

``` shell
php artisan jwt:generate
```

it prints this on the terminal screen

``` shell
jwt-auth secret [63h9P6G33YbO1QeQjXnd4XW1HymQV5RN] set successfully.
```

But didn't replace the the default value in the .env file

Note: even if I leave it empty it doesn't place it!!

**Comment by thelastblack** [comment]
I have encountered the bug too.

**Comment by wstaples** [comment]
I'm encountering this as well. I noticed that the published config is not being updated either.

**Comment by Mahmoudz** [comment]
I submitted a PR that fixes this #1050
